go.mod

```diff
@@ -11,7 +11,7 @@ replace (
 require (
 	github.com/Masterminds/sprig/v3 v3.2.2
 	github.com/apparentlymart/go-cidr v1.0.1
-	github.com/aws/aws-sdk-go v1.38.65
+	github.com/aws/aws-sdk-go v1.44.106
```

Comment:
did you update this? I don't think it has anything to do with tigera operator or?

Reply:
I didn't update it manually but I will check why `go mod tidy` changed it.